clients/fides-js/src/components/Overlay.tsx

```diff
@@ -140,6 +140,9 @@ const Overlay: FunctionComponent<Props> = ({
 
   return (
     <div>
+      {bannerIsOpen && window.Fides.options.preventDismissal && (
+        <div className="fides-modal-overlay" />
```

Comment:
Instead of the `useEffect` in ConsentBanner, would it work to give this div an `onClick` which closes the banner? I generally try to avoid `ref`s as I find they can get confusing